Source/WebCore/page/Quirks.cpp

```diff
@@ -1859,4 +1856,15 @@ bool Quirks::needsGetElementsByNameQuirk() const
 #endif
 }
 
+bool Quirks::needsNetflixMediaSessionQuirk() const
```

Comment:
1. Open a bugs.webkit.org to remove this quirk in the future [Example](https://bugs.webkit.org/show_bug.cgi?id=270360)
2. Add a comment with the reference of the bug, in order that people who needs to test (QA, etc.) why this quirk has been done, they can have access to steps to reproduce.